applications/DEMApplication/custom_constitutive/DEM_KDEM_with_damage_parallel_bond_CL.cpp

```diff
@@ -549,7 +549,7 @@ namespace Kratos {
 
         // HERE IT STARTS THE UNBONDED PART
         double OldUnbondedLocalElasticContactForce[2] = {0.0};
-        double UnbondedLocalElasticContactForce[2] = {0.0};
+        double UnbondedLocalElasticContactForce[3];
```

Comment:
I do not understand this change. We never access a third component. And on top of that, you are now not initializing the value.